Sandbox/demo-scichart-angular/src/app/app.component.ts

```diff
@@ -0,0 +1,438 @@
+import { Component } from '@angular/core';
+import { ScichartAngularComponent } from 'scichart-angular';
+import { appTheme } from './theme';
+import {
+  SciChartSurface,
+  NumericAxis,
+  FastLineRenderableSeries,
+  XyDataSeries,
+  EllipsePointMarker,
+  SweepAnimation,
+  SciChartJsNavyTheme,
+  NumberRange,
+  MouseWheelZoomModifier,
+  ZoomPanModifier,
+  ZoomExtentsModifier,
+  EAxisType,
+  ESeriesType,
+  EChart2DModifierType,
+  Vector3,
+  CameraController,
+  NumericAxis3D,
+  zeroArray2D,
+  UniformGridDataSeries3D,
+  GradientColorPalette,
+  SurfaceMeshRenderableSeries3D,
+  EDrawMeshAs,
+  FastBandRenderableSeries,
+  XyyDataSeries,
+  MouseWheelZoomModifier3D,
+  OrbitModifier3D,
+  ResetCamera3DModifier,
+  SciChart3DSurface,
+  FastBubbleRenderableSeries,
+  XyzDataSeries,
+  SplineLineRenderableSeries,
+  TextAnnotation,
+  ECoordinateMode,
+  EHorizontalAnchorPoint,
+  EVerticalAnchorPoint,
+  SeriesAnimation,
+  WaveAnimation,
+  ScaleAnimation,
+  FadeAnimation,
+  GenericAnimation,
+} from 'scichart';
+
+const waveAnimation = new WaveAnimation({
+  zeroLine: 0,
+  pointDurationFraction: 0.5,
+  duration: 1000,
+  fadeEffect: true,
+});
+const sweepAnimation = new SweepAnimation({ duration: 1000 });
+const scaleAnimation = new ScaleAnimation({ duration: 1000, zeroLine: 0 });
+const fadeAnimation = new FadeAnimation({ duration: 1000 });
+
+SciChartSurface.loadWasmFromCDN();
+SciChart3DSurface.loadWasmFromCDN();
```

Comment:
TODO for myself @jim-risen. We also need to demonstrate how to serve .wasm and .data files instead of using them from CDN.